app/javascript/pages/admin/Reporting/index.jsx

```diff
@@ -0,0 +1,208 @@
+import React, { useContext, useState, useEffect } from 'react'
+import * as R from 'ramda'
+import moment from 'moment'
+import { useTranslation } from 'react-i18next'
+import { addDays } from 'date-fns'
+import styled from 'styled-components'
+import { useLazyQuery } from '@apollo/react-hooks'
+
+import {
+  Dropdown,
+  Menu,
+  Item,
+  Field,
+  Select
+} from '@zendeskgarden/react-dropdowns';
+import { Field as FormField, Label as FormLabel, Input } from '@zendeskgarden/react-forms'
+import { Datepicker } from '@zendeskgarden/react-datepickers'
+import { Button } from '@zendeskgarden/react-buttons'
+import { Skeleton } from '@zendeskgarden/react-loaders';
+
+import UserReporting from './User'
+import ReportingQuery from './queries/userReportingQuery.gql'
+import { FilterContext, officeFilterValueLens } from '/context'
+import OfficeFilter from '../../../components/OfficeFilter'
+
+const AdminSection = styled.div`
+  display: flex;
+  justify-content: space-around;
+  padding-bottom: 50px;
+`
+
+const ToolbarHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid ${({theme}) => theme.palette.grey["200"]};
+  padding-bottom: 16px;
```

Comment:
should we be using spacing for this or was the intention for it to be a constant 16px?